src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -288,11 +300,12 @@ export class UBSOrderDetailsComponent implements OnInit, OnDestroy {
       cert.certificateStatus === 'NEW'
     ) {
       this.certificateSum = this.certificateSum + cert.certificatePoints;
-      this.certMessage = `Сертифiкат на cуму ${cert.certificatePoints} грн активовано. Строк дії сертифікату - до ${cert.certificateDate}`;
+      this.certMessage = this.certMessageFirst + ' ' + cert.certificatePoints +
+       ' ' + this.certMessageFourth + ' ' + cert.certificateDate;
       this.displayCert = true;
     } else if (cert.certificateStatus === 'USED') {
```

Comment:
maybe, will be better put this magic strings into some enum?

Reply:
I'm not sure that it will be good in this case

Reply:
agree with Yurii... you need to create enum with certificate Statusses

Reply:
Done

Reply:
Done